### DISCUSSION TITLE
Styling in Select

### DISCUSSION THREAD

**Discussion by hanan-commits-work** [iTwin/itwinjs-core#3063] [Q&A]
Hi.
I am having problem in changing the 'Box Size' of the Select Drop down menu. I can change the width, but not lower the box size. When I changed height, it changed my placement of the box to down or up. I tried using Padding, Borders, but all in vain.
Kindly Help

**Comment by NancyMcCallB** [comment]
Are you  using the Select component from itwinui-react? It included a menuStyle prop for custom menu styling.
